packages/core/src/interpreter.ts

```diff
@@ -477,13 +479,8 @@ export class Interpreter<
    *
    * @param event The event(s) to send
    */
-  public send: PayloadSender<TEvent> = (event, payload?): void => {
-    const eventObject = toEventObject(event, payload);
-    const _event = toSCXMLEvent(eventObject);
-    // console.log(
-    //   `${_event.origin?.sessionId} -> ${this.sessionId}`,
-    //   _event.name
-    // );
+  public send(event: TEvent | SCXML.Event<TEvent>) {
```

Comment:
Note: IMHO we should also consider removing `SCXML.Event` too - it breaks subtyping and stuff. That, of course, should be done in a separate PR and all.

Reply:
The slight problem with that is that other actors should send SCXML events to actors, since they contain `origin` data. There are probably other ways to deal with the typing issue